dockers/docker-teamd/Dockerfile.j2

```diff
@@ -17,6 +17,22 @@ RUN apt-get update
 {{ install_debian_packages(docker_teamd_debs.split(' ')) }}
 {%- endif %}
 
+# W/A: install python2
+RUN apt-get -y --no-install-recommends install python2.7 python-pip
+
+# W/A: upgrade pip via PyPI and uninstall the Debian version
+RUN pip2 install --upgrade pip
+
+# W/A: setuptools and wheel are necessary for installing some Python wheel packages
+RUN pip2 install --no-cache-dir setuptools==44.1.1
+RUN pip2 install --no-cache-dir wheel==0.35.1
+
+# W/A: install supervisor
+RUN pip2 install supervisor==4.2.1
+
+# W/A: add support for supervisord to handle startup dependencies
+RUN pip2 install supervisord-dependent-startup==1.4.0
```

Comment:
Can we combine into single RUN instruction to reduce build time?

Reply:
@stepanblyschak this pattern has been taken from the original Makefile and it has some logical blocks separation. Not sure if we can gain much performance from combining install steps.

Reply:
It does make a difference: https://github.com/Azure/SONiC/blob/master/doc/sonic-build-system/build_system_improvements.md#improving-dockerfile-instructions.

We do not gain anything from seperate RUN instructions since cache is disabled.